apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -186,6 +186,12 @@ const Payment = new GQLListSchema('Payment', {
             defaultValue: PAYMENT_INIT_STATUS,
         },
 
+        order: {
+            schemaDoc: 'Payment order number',
+            type: Text,
```

Comment:
`Text`? Why not `Integer`?

Reply:
Not sure if payment orders will contain special chars or not - dots, slashes .... And it's not really important to have number field here, as we are not going to make math operations on database layer with this field